src/data/markdown/translated-guides/en/02 Using k6/05 k6 Options/02 Reference.md

```diff
@@ -496,7 +496,7 @@ $ k6 run --include-system-env-vars ~/script.js
 
 A boolean, true or false. When this option is enabled (set to true), all of the verifications that
 would otherwise be done to establish trust in a server provided TLS certificate will be ignored.
-This only applies to connections created by VU code, such as http requests.
+This only applies to connections created from code, such as HTTP requests.
```

Comment:
```suggestion
This only applies to connections created from a script, such as HTTP requests.
```

Suggestion, I'm wondering if saying "from a script" vs "from code" reads better. Feel free to ignore. 🙏

Reply:
TBH, I feel like `from code` here is more appropriate. @mstoykov WDYT?

Reply:
I also vote for `code` based on the very picky part of `script` being a thing within ECMAScript/JS and there also being `modules`. 

So this  new version *can* be read as this iwll only works in `script` but not `modules.